packages/delegation-process/package.json

```diff
@@ -5,8 +5,7 @@
   "main": "dist",
   "type": "module",
   "scripts": {
-    "test": "vitest",
-    "test:it": "vitest integration",
+    "test": "vitest --config vitest.integration.config.ts",
```

Comment:
This kept as `test` rather than `test:integration` since `turbo.json` isn't updated yet. I can rebase and rename this after https://github.com/pagopa/interop-be-monorepo/pull/1758 is merged.